src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -1477,6 +1562,16 @@ public static string WildCardRecursiveCannotBePartOfPath(string wildCardRec, str
       }
 
       public static string IndividualWithIdNotFound(int individualId) => $"Individual with id `{individualId}` not found.";
+
+      public static string TypeNotSupported(Type t)
+      {
+         return $"{t} is not currently been handled";
```

Comment:
@msevestre, This is used when switching on types for the DataFormatParameter. So, in some places we have different logic depending on the parameter type. Consider creating the options for a given row on the grid when clicking on the excel column cell. The options are all of the same type (i.e., if the row is a mapping, all options are mappings, if meta data all options are also meta data...). The way to create these options depends on the source object of the row so we switch logic based on the type of the source. Currently we have a default option that should not be hit since all possible options for the source type are listed in the cases. In the future there could be a new type but we could forget to add handling such a type here in the switch so the default will be hit showing the error type not supported. This is the context where the TypeNotSupported method is used.
I agree it could expect just the name of the type. Passing a string allows for any string though so the caller could pass the string he/she builds with some error and still this method will show the string passed. This is why it was a type, to ensure it is properly used. I changed it to string and rename it to "typeName" but I do not completely like it like that since again, it allows for non-intentional wrong use but I could live with it since it is more an error targeting developers than final users.

Reply:
Type.ToString() will return the name with the namespace
For instance `t="OSPSuite.Core.Domain.UnitSystem.Unit"`. I don't think this is a message we want to show to the user
So my suggestion is to have one method call the other passing type.name as parameter
And then correct the grammar